models/event.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const Build = require('./build');
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Pipeline = require('./pipeline');
+const Scm = require('../core/scm');
+const Workflow = require('../config/workflow');
+
+const buildId = Joi.reach(Build.base, 'id');
+
+const MODEL = {
```

Comment:
Do we want to have overall status?  Did the event complete successfully?  Or do we just want to infer that from all builds?